sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -95,7 +138,13 @@ def fillna(self, value, method, axis, **kwargs):
   def ffill(self, **kwargs):
     return self.fillna(method='ffill', **kwargs)
 
+  @frame_base.args_to_kwargs(pd.DataFrame)
+  @frame_base.populate_defaults(pd.DataFrame)
+  def bfill(self, **kwargs):
+    return self.fillna(method='bfill', **kwargs)
```

Comment:
This method is order sensitive, right?

Reply:
Yep! fillna will throw for this: https://github.com/apache/beam/blob/b787dd1e252cbb9f8684ad0e1428bf7a2419e164/sdks/python/apache_beam/dataframe/frames.py#L76-L78

I followed your example from ffill.